test/12_nullablearray.jl

```diff
@@ -970,8 +970,18 @@ for ordered in (false, true)
     end
 end
 
+# Test vcat with nulls
 @test isequal(vcat(NullableCategoricalArray(["a","b"],[false,true]),
            NullableCategoricalArray(["b","a"],[true,false])),
            NullableCategoricalArray(["a","","","a"],[false,true,true,false]))
 
+# vcat with all nulls preserves isordered
+ca1 = NullableCategoricalArray(["a","b"],[false,true])
+ordered!(ca1, true)
+ca2 = NullableCategoricalArray(["a","b"],[true,true])
+ordered!(ca2, false)
+r = vcat(ca1,ca2)
+@test isequal(r, NullableCategoricalArray(["a","","",""],[false,true,true,true]))
+@test isordered(r)
```

Comment:
The ideas is that concatenation of an array with only nulls shouldn't affect the ordering. But I'll postpone that to another PR.